### DISCUSSION TITLE
Using phone's Bluetooth speaker and microphone

### DISCUSSION THREAD

**Discussion by FGameCZ** [opencardev/crankshaft#600] [Q&A]
Hi,
is it possible to use speaker and microphone connected to phone via Bluetooth? I was using Headunit Reloaded and it had an option to play the sound from the phone's speaker.

**Comment by Gavfo** [comment]
I use the headunit in my car connected via Bluetooth to my phone for sound , no problem.
Waze and music content are controlled from AA running on my PI. Seems to work great.
